<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>box1</string>
    </property>
    <widget class="QPushButton" name="showButton">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>270</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SHOW</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxModel">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>270</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Любые</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>model1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>model2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelModel">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Модель</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxVehicleType">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>270</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentText">
      <string>Любые</string>
     </property>
     <item>
      <property name="text">
       <string>Любые</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Легковые</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Мотоциклы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Грузовики</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxMark">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Любые</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelMark">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Марка:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelVehicleType">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>240</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип транспорта</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>30</y>
       <width>161</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:40px;
border: 2px solid red;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>420</y>
       <width>1351</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="vehicleGrid" rowstretch="0,0,0,0" columnstretch="0,0,0,0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:64px</string>
        </property>
        <property name="text">
         <string>БИПКИ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="showAddGroupBox">
     <property name="geometry">
      <rect>
       <x>1410</x>
       <y>40</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="addGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>box2</string>
    </property>
    <widget class="QComboBox" name="comboBoxVehicleTypeAddBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>270</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentText">
      <string>Не выбрано</string>
     </property>
     <item>
      <property name="text">
       <string>Не выбрано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Легковая</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Мотоцикл</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Грузовик</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxMarkAddBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Не выбрано</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="addObjectConfirmed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1460</x>
       <y>720</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="returnToMainPage">
     <property name="geometry">
      <rect>
       <x>1410</x>
       <y>40</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>На главную</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelVehicleTypeOnAddPage">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>240</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип транспорта</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMarkOnAddPage">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Марка:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="inputGroupBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>641</width>
       <height>431</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
font-style:Arial;
background-color: #f2f2f2;
border: 2px solid #4CAF50;
border-radius: 5px;
padding: 5px;
}</string>
     </property>
     <property name="title">
      <string>Заполните поля:</string>
     </property>
     <widget class="QLabel" name="labelImage">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите&lt;br/&gt;изображение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectImageButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>+img</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabelName">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>270</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Файл не выбран&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelImageButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>140</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
